docs/dbus/DBusAPIReference.lyx

```diff
@@ -405,9 +405,9 @@ devices device nodes of devices to form the pool.
 
 \end_deeper
 \begin_layout Description
-Returns: the object path of the constructed pool and the device nodes of
- the blockdevs added to the pool.
- Signature: (oas)qs.
+Returns: the object path of the constructed pool and the object path and
+ device nodes of the blockdevs added to the pool.
+ Signature: oa(os)qs.
```

Comment:
Maybe better just to have object path, and to drop devnode part for this and the one below.

Reply:
sounds good.